package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "license": "BSD-3",
   "scripts": {
-    "dev": "nuxt dev",
+    "dev": "nuxt dev --dotenv .env.test",
```

Comment:
Nomenclature wise - any chance we could call this environment and associated keys `DEV` instead of `TEST`? Reason being -- to my mind `TEST` is explicitly a QA stage, wherein things like integ tests are run, whereas with this change we specifically want a database to develop against.

What do you think?

Reply:
ahhhh totally fair question! I wrestled with this before I made it. I was choosing between QA, DEV, and TEST for building a new environment. I went with TEST since we probably will be running tests against this environment, and if we might have different api's/functions in this environment. (more than just DB actually)

I'm thinking it would be best to merge for now regardless. and then we could change later if we want. 

What do you think?

Reply:
Sounds good. Let's decide post-facto. Since it's being picked up automagically when running `yarn dev` and contributors aren't making explicit changes for it (nor should they need to be aware of it) we can switch it in the background without issue I think.